dnplab/dnpTools.py

```diff
@@ -614,3 +615,20 @@ def zero_fill(
         all_data[all_data.processing_buffer] = data
     else:
         return data
+
+
+def voigtian(x, amp, x0, sigma, gamma):
+    """
+    Voigt is a combintaion of Gaussian and Lorentzian lineshapes
+    """
+    z = ((x0 - x) + 1j * gamma) / (sigma * np.sqrt(2.0))
+    fit = amp * np.real(wofz(z))
```

Comment:
Remove the amplitude from the function
```suggestion
    fit = np.real(wofz(z))
```